### PR TITLE
openlrs: add simple UI

### PR DESCRIPTION
**Pull request by mlyle**
* Reconstruct vector art for left-side config icons
* Eliminate some last dead code in openlrs
* Move important parameter fields to enums
* Build a UI screen

### CODE REVIEW

**Comment by mlyle** [comment]
image

**Comment by LitterBugs** [comment]
Completed initial bench testing with two sets of OPlinks/Revolutions.  Was able to bind after setting TX/RX respectively.  Was able to remotely connect to the revolution over the link from GCS connected to the oplink.  

Will do some additional testing of the beacon functionality, range testing, flight testing, etc.

**Comment by LitterBugs** [comment]
Field tested at short range.  Link appears stable.  Tested beacon functionality with handheld and SDR.

Samples from  SDR testing
dropenlrstesting
dropenlrsbeacon

File: ground/gcs/src/plugins/config/configradiowidget.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+void ConfigRadioWidget::resizeEvent(QResizeEvent *event)
+{
+    QWidget::resizeEvent(event);
+}
+
+void ConfigRadioWidget::enableControls(bool enable)
+{
+    Q_UNUSED(enable);
```
Comment (tracernz) [inline ground/gcs/src/plugins/config/configradiowidget.cpp:70]: Should call ConfigTaskWidget::enableControls

File: ground/gcs/src/plugins/config/configradiowidget.h [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    ~ConfigRadioWidget();
+
+private slots:
+
+private:
+    Ui::Radio *ui;
+
+protected:
+    void resizeEvent(QResizeEvent *event);
+    virtual void enableControls(bool enable);
```
Comment (tracernz) [inline ground/gcs/src/plugins/config/configradiowidget.h]: Should mark `override`

File: ground/gcs/src/plugins/config/configradiowidget.cpp [hunk 3]
```diff
@@ -61,6 +70,61 @@ void ConfigRadioWidget::enableControls(bool enable)
...
+{
+    (void) obj;
+
+    OpenLRS *olrsObj = OpenLRS::GetInstance(getObjectManager());
+
+    OpenLRS::DataFields openlrs = olrsObj->getData();
+
+    switch (openlrs.role) {
+    case OpenLRS::ROLE_DISABLED:
+    default:
```
Comment (tracernz) [inline ground/gcs/src/plugins/config/configradiowidget.cpp:99]: Shouldn't be too hard for ConfigTaskWidget to do radio buttons (if it doesn't already)... out of scope for this PR if you like.

File: ground/gcs/src/plugins/config/configradiowidget.h [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    Q_OBJECT
+
+public:
+    ConfigRadioWidget(QWidget *parent = nullptr);
+    ~ConfigRadioWidget();
+
+private slots:
+    void roleChanged(bool);
+    void refreshWidgetsValues(UAVObject *obj);
+    void updateObjectsFromWidgets();
```
Comment (tracernz) [inline ground/gcs/src/plugins/config/configradiowidget.h:50]: `override` x2

**Review by tracernz — APPROVED** [review]
LGTM

**Pull request merged**